build/automation/lib/aws.mk

```diff
@@ -236,7 +236,14 @@ aws-rds-describe-instance: ### Describe RDS instance - mandatory: DB_INSTANCE
 		$(AWSCLI) rds describe-db-instances \
 			--region $(AWS_REGION) \
 			--db-instance-identifier=$(DB_INSTANCE) \
-	" | make -s docker-run-tools CMD="jq -r '.DBInstances[0]'"
+	" | make -s docker-run-tools CMD="jq -r '.DBInstances[0].Endpoint.Address'"
```

Comment:
Is this line expected to be changed?

Reply:
good spot missed it, made the change to test the function added before creating it 👍